xtensa-host.sh

```diff
@@ -1,7 +1,7 @@
 if [ $# -lt 1 ]
 then
   echo "usage: $0 device [-k kernel] [-t] [-d] [-i] [-r rom] [-c] [-g] [-o time log]"
-  echo "supported devices: byt, cht, hsw, bdw, bxt, sue, cnl, icl, skl, kbl, hky, tgl"
+  echo "supported devices: byt, cht, hsw, bdw, bxt, sue, cnl, icl, skl, kbl, hky, tgl, imx8, imx8x, imx8m"
   echo "[-k] | [--kernel]: load firmware kernel image"
```

Comment:
This doesn't look good. I think we should add imx8, imx8x in a separate patch. 

Commit should say: xtensa-host: Update xtensa-host.sh usage for imx8, imx8x. And then the current patch will only contain the addition of imx8m.